### PR TITLE
PROD-345: deck completion height issue.

### PR DESCRIPTION
**Pull request by yash431garg**
- Description
Changes no question card height to a fix height 350px with overflow scroll.
- Screen shots or recordings for UI changes
Screenshot 2024-09-27 at 1 33 39 PM

### CODE REVIEW

**Comment by marvinmarnold** [comment]
@neel-ds can you review

File: app/components/NoQuestionsCard/NoQuestionsCard.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export function NoQuestionsCard({
 
   return (
     <div className="flex flex-col justify-between h-full w-full gap-4">
-      <div className="bg-gray-700 min-h-[350px] w-full max-w-[480px] rounded-xl pt-6 pl-4 pr-4 flex flex-col justify-between border border-gray-500 text-white relative mb-[4px]">
+      <div className="bg-gray-700 h-[350px] w-full max-w-[480px] rounded-xl pt-6 pl-4 pr-4 flex flex-col justify-between border border-gray-500 text-white relative mb-[4px] overflow-scroll">
```
Comment (marvinmarnold) [inline app/components/NoQuestionsCard/NoQuestionsCard.tsx:32]: Why did you just change NoQuestionsCard? I think all of the deck completion screens are in scope.

**Review by marvinmarnold — CHANGES_REQUESTED** [review]

**Comment by neel-ds** [comment]
@yash431garg Apply same changes to following files:
https://github.com/gator-labs/chomp/blob/5b302c1e599ab031d8349d4474efb6991fa063f5/app/components/PreviewDeckCard/PreviewDeckCard.tsx#L27

https://github.com/gator-labs/chomp/blob/5b302c1e599ab031d8349d4474efb6991fa063f5/app/components/QuestionCard/QuestionCard.tsx#L75

**Comment by yash431garg** [comment]
> @yash431garg Apply same changes to following files:
> 
> https://github.com/gator-labs/chomp/blob/5b302c1e599ab031d8349d4474efb6991fa063f5/app/components/PreviewDeckCard/PreviewDeckCard.tsx#L27
> 
> https://github.com/gator-labs/chomp/blob/5b302c1e599ab031d8349d4474efb6991fa063f5/app/components/QuestionCard/QuestionCard.tsx#L75

Thanks @neel-ds, I just added those changes.

File: app/components/NoQuestionsCard/NoQuestionsCard.tsx (hunk 1 above)
Comment (yash431garg) [reply app/components/NoQuestionsCard/NoQuestionsCard.tsx:32]: Just added them. I thought only deck completion card was in the scope.

**Review by neel-ds — APPROVED** [review]

**Pull request merged**